tests/integration/components/public/copyright-item-test.js

```diff
@@ -0,0 +1,10 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('public/copyright-item', 'Integration | Component | copyright item');
```

Comment:
The module name should also be `Integration | Component | public/copyright item`